### DISCUSSION TITLE
Show Running Tasks

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#67] [Course Qeustions] [answered]
Hi Mair,
A little off topic.
Was hoping you might know how to chage the default behaviour of the "show running tasks" function in VSCODE using esp-idf extension?
While compiling, the output does not appear in the Terminal window unless I click the "Show Running Tasks" in VSCODE taskbar (as shown below). Then it displays the compilation output. However, next time I compile, the output is again off and I have to click this icon every time I buiold if I want to see output.

![taskbar](https://user-images.githubusercontent.com/41096656/161845135-d1ff7806-d756-4b99-9ae6-b491b20e6c31.JPG)

**Comment by Mair** [comment, marked as answer]
hi @orbitcoms, unfortunately not that I know of. This is one of the reasons I prefer to use the idf command line
